data/mods/Xedra_Evolved/eocs/item_eocs.json

```diff
@@ -23,5 +23,27 @@
     },
     "effect": [ { "u_message": "local_files_xedra", "snippet": true, "same_snippet": true, "popup": true } ],
     "false_effect": [ { "u_message": "You found nothing interesting." } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_WOLF_SLEDGE_TRHOW",
+    "condition": { "compare_num": [ { "u_val": "stamina" }, ">=", { "const": 3000 } ] },
+    "effect": [
+      {
+        "u_run_inv_eocs": "all",
+        "search_data": [ { "id": "mace_magic", "wielded_only": true } ],
+        "true_eocs": [
```

Comment:
should nested false_eocs be added if player tries to activate weapon while it's in inventory, that u_message "can't throw while in inventory!", or no message is fine?

Reply:
Probably yeah